### DISCUSSION TITLE
Need example configuration for enabling distributed tracing to Datadog

### DISCUSSION THREAD

**Discussion by nvenkatp** [ory/hydra#2564] [Q&A] [answered]
Hello,

I am looking for hydra configuration to enable distributed tracing to Datadog. We had setup hydra in our AWS beanstalk env and we already using Datadog for other services and so would like to use the same for this tracing.

Could someone point me the example configuration on how to do that ?

**Comment by vinckr** [comment, marked as answer]
Hey, 
there is an example configuration included in the quickstart: 
https://github.com/ory/hydra/blob/master/quickstart-tracing.yml#L58-L63
Just uncomment the Datadog bits. 
https://www.ory.sh/hydra/docs/guides/tracing/

Let me know if that was helpful 🐝

**Reply by vinckr** [reply]
Hey @nvenkatp ,
Just checking in if your question was answered or if there was still something unclear.
I will mark this as answered for now, but feel free to come back to the discussion any time.
